packages/db/postgres/tables.sql

```diff
@@ -260,3 +280,19 @@ CREATE TABLE IF NOT EXISTS pinning_authorization
   inserted_at     TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL,
   deleted_at      TIMESTAMP WITH TIME ZONE
 )
+
+CREATE VIEW admin_search as
+select
+  u.id::text as user_id,
+  u.github_id as username,
```

Comment:
```suggestion
  u.github as username,
```

Not sure if this is what you want, just to make sure. user.github is a numeric github ID, not the handle. We don't track the handle in web3.storage

Reply:
removed this field.